torch/onnx/symbolic_opset9.py

```diff
@@ -114,6 +114,12 @@ def div(g, self, other, *args):
         return _div_rounding_mode(g, self, other, *args)
 
 
+@parse_args("v", "v", "v", "f")
```

Comment:
Can we use `t` instead of `v` for the first 3 args to enforce / express that they're tensor values?

Reply:
Based on most other symbolics, seems like parse_args("v") is used for input tensors.